configs/full.yaml

```diff
@@ -99,10 +92,27 @@ npz_fixed_field_keys:
 # key_mapping:
 #   free_energy: total_energy
 
+# npz example 
+# the keys used need to be stated at least once in key_mapping, npz_fixed_field_keys or include_keys
+# key_mapping is used to map the key in the npz file to the NequIP default values (see data/_key.py)
+# all arrays are expected to have the shape of (nframe, natom, ?) except the fixed fields
+# note that if your data set uses pbc, you need to also pass an array that maps to the nequip "pbc" key
+dataset: npz                                                                       # type of data set, can be npz or ase
+dataset_url: http://quantum-machine.org/gdml/data/npz/toluene_ccsd_t.zip           # url to download the npz. optional
+dataset_file_name: ./benchmark_data/toluene_ccsd_t-train.npz                       # path to data set file
+key_mapping:
+  z: atomic_numbers                                                                # atomic species, integers
+  E: total_energy                                                                  # total potential eneriges to train to
+  F: forces                                                                        # atomic forces to train to
+  R: pos                                                                           # raw atomic positions
+npz_fixed_field_keys:                                                              # fields that are repeated across different examples
+  - atomic_numbers
+
 # A list of atomic types to be found in the data. The NequIP types will be named with the chemical symbols, and inputs with the correct atomic numbers will be mapped to the corresponding types.
 chemical_symbols:
- - H
- - C
+  - H
+  - C
+
 # Alternatively, you may explicitly specify which chemical species maps to which type in NequIP (type index; the name is still taken from the chemical symbol)
```

Comment:
@Linux-cpp-lisp I think this is a bit unclear what exactly a "type index" is here and what is meant by "name is still taken from the chemical symbol"

Reply:
```suggestion
# Alternatively, you may explicitly specify which chemical species in the input will map to NequIP atom type 0, which to atom type 1, and so on.
# Other than providing an explicit order for the NequIP atom types, this option behaves the same as `chemical_symbols`
```